pkg/services/hover.go

```diff
@@ -12,7 +12,7 @@ import (
 func Hover(params protocol.HoverParams, cache *utils.Cache, context *utils.LsContext) (protocol.Hover, error) {
 	doc, err := yamlparser.ParseFromUriWithCache(params.TextDocument.URI, cache, context)
 	if err != nil {
-		return protocol.Hover{}, nil
+		return protocol.Hover{}, err
```

Comment:
i dont think showing an error when hovering is very good, that's why it was returning nil